Package@swift-5.5.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Time",
+    products: [
+        .library(name: "Time", targets: ["Time"])
+    ],
+    dependencies: [
+        
+    ],
+    targets: [
+        .target(name: "Time", dependencies: []),
+        
+        .testTarget(name: "TimeTests", dependencies: ["Time"]),
+    ]
+)
```

Comment:
Is this `Package.swift` necessary if the primary one is set to request Swift 5.5?

Reply:
Pre 5.5, the compiler emits a warning of unused/unrecognized files and suggests explicitly excluding the DocC file.

To be honest, I’m not 100% sure if that warming is emitted when downstream users are building the library as a dependency, but it definitely emits a warning when building the project itself

Reply:
I just tested this by depending on a special branch that doesn't do this conditional package manifest stuff and I can conform that the compiler warning _does_ get emitted when the library is built by a downstream user.

> found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
> /Users/.../Library/Developer/Xcode/DerivedData/test-crmhlaksfvpqexbqodizljvczqrs/SourcePackages/checkouts/time/Sources/Time/Documentation.docc

Reply:
Also, if your Package.swift doesn't somewhere explicitly say 5.5 is a supported build flavor, the DocC output is not generated

Reply:
Oh, shoot. I'm seeing my mistake - I didn't properly switch the minimum version in question... I'll fix it!